custom_components/ai_thermostat/models/utils.py

```diff
@@ -58,7 +40,4 @@ def temperature_calibration(self):
 	if new_calibration > float(self._max_temp):
 		new_calibration = float(self._max_temp)
 	
-	#loop = asyncio.get_event_loop()
```

Comment:
This is not part of the PR's intended changes. Please remove this from your changeset :)